Source/Core/Plugins/EventProcessor/Default/20_ErrorPlugin.cs

```diff
@@ -39,6 +39,9 @@ public class ErrorPlugin : EventProcessorPluginBase {
 
             error.Data[Error.KnownDataKeys.TargetInfo] = targetInfo;
 
+            if (context.StackSignatureData.Count > 0)
```

Comment:
Hmm, I don't see a project for Exceptionles.Core tests, which is where I made my changes.